src/server/AppManager.ts

```diff
@@ -733,6 +735,31 @@ export class AppManager {
         return this.enableApp(storageItem, app, true, isManual, silenceStatus);
     }
 
+    private async installApp(storageItem: IAppStorageItem, app: ProxiedApp, user: IUser): Promise<boolean> {
+        let result: boolean;
```

Comment:
Maybe lose this variable and return a boolean directly in the `try/catch` block? We won't run any code after it, after all. Does it make sense?

Reply:
Let's review this during the beta period